### DISCUSSION TITLE
Aliases and the development roadmap

### DISCUSSION THREAD

**Discussion by Idobenhamo** [artempyanykh/marksman#252] [General]
Hi,
Any chance of supporting aliases for files?
Do you plan on keeping the development of this?
Thanks a lot!

**Comment by artempyanykh** [comment]
Hi @Idobenhamo! Thanks for asking. 

> Any chance of supporting aliases for files?

Could you elaborate on what you mean by "aliases for files"?

> Do you plan on keeping the development of this?

Well, I do 😄 With that said, I'd be happy to accept contributions. #258 is a good example of scratching your own itch.

**Reply by Idobenhamo** [reply]
Thanks!
I would try to learn F# to try to help with this amazing project, thank you very much!

By aliases I mean if a file for example called `temp_long_name.md` you can define alias in the file that called `temp` and you can link this file by using the alias.

Also, There is any explanation on how to use the new hashtag features? and what with the feature of supporting non md files?

again, thanks a lot for everything!! @artempyanykh
